### DISCUSSION TITLE
Unable to load module on Emacs 29.4

### DISCUSSION THREAD

**Discussion by sivaramn** [minad/jinx#207] [Q&A]
when I build jinx within the elpa folder and open enable jinx-mode I get the following error

`Debugger entered--Lisp error: (module-open-failed "c:/Users/sivar/AppData/Roaming/.emacs.d/elpa/jinx-1.10/jinx-mod.dll" "The specified module could not be found.")

  module-load("c:/Users/sivar/AppData/Roaming/.emacs.d/elpa/jinx-1.10/jinx-mod.dll")

  jinx--load-module()

  jinx-mode(toggle)

  funcall-interactively(jinx-mode toggle)
  command-execute(jinx-mode record)
  execute-extended-command(nil "jinx-mode")
  amx-read-and-run(((getenv . 5) (occur . 1) (find-grep . 1) (ediff-files . 6) (shell . 16) (dired-omit-mode . 1) (pdf-view-mode . 1) (ffap . 5) (ansi-term . 4) (term . 3) (pdf-tools-install . 3) (shell-strip-ctrl-m . 2) (pdf-tools-enable-minor-modes . 2) (dired . 1) (pdf-outline . 1) (package-install . 1) (comint-strip-ctrl-m . 1) (toggle-debug-on-error . 1) (set-terminal-coding-system . 1) (cd) (5x5) (amx) (arp) (dbx) (dig) (erc) (ert) (eww) (ftp) (gdb) (irc) (jdb) (man) (mpc) (pdb) (pwd) (rsh) (sdb) (xdb) (calc) (diff) (dirs) (gnus) (grep) (help) (ielm) (info) (life) (mail) (mpuz) ...))
  amx()
  funcall-interactively(amx)
  command-execute(amx)
`

on msys2 I compiled it like this
`/c/Users/sivar/AppData/Roaming/.emacs.d/elpa/jinx-1.10
$ x86_64-w64-mingw32-gcc -I. -O2 -Wall -Wextra -fPIC -shared -o jinx-mod.dll jinx-mod.c -IC:/msys64/mingw64/include/enchant-2 -IC:/msys64/mingw64/include -IC:/msys64/mingw64/include/glib-2.0 -IC:/msys64/mingw64/lib/glib-2.0/include -LC:/msys64/mingw64/lib -lenchant-2
`

This creates a dll but is not being recognized by `GNU Emacs 29.4 (build 2, x86_64-w64-mingw32) of 2024-07-05`

what am I doing incorrectly?

**Comment by minad** [comment]
Hi! I am not familiar with using Jinx on Windows. You can search the issue tracker/discussion board. There might have been similar compilation issues on Windows discussed before.
